client/src/components/PacmanGame/PacmanGame.jsx

```diff
@@ -16,25 +17,41 @@ import PacmanBoard from './PacmanBoard';
 class PacmanGame extends Component {
   state = {
     pacman: {
-      x: 0,
+      x: 1,
       y: 5,
+      direction: 'RIGHT',
     },
-    direction: 'RIGHT',
+    ghosts: [
+      {
+        direction: 'RIGHT',
+        x: 19,
+        y: 1
+      },
+      {
+        direction: 'UP',
+        x: 1,
+        y: 19
+      },
+      {
+        direction: 'DOWN',
+        x: 23,
+        y: 5
+      },
+      {
+        direction: 'LEFT',
+        x: 23,
+        y: 23
+      }
+    ],
     score: 0,
+    status: 0, // 0 - Not-started, 1 - Progress, 2 - Finished, 3 - Paused
```

Comment:
I didn't do that because key-value type objects we will be needing. Instead, we can combine entityToCode, codeToEntity, etc into a closure, and the logic should be from gameCore.js.